codeJE/workflows.yml

```diff
@@ -0,0 +1,53 @@
+---
+options:
+  global: ""
+  local:
+    - "-b"
+    - "--configuration json://$JSON"
+    - "--aod-memory-rate-limit 2000000000"
+    - "--shm-segment-size 16000000000"
+    - "--min-failure-level error"
+
+workflows:
+  # QA
+
+  o2-analysis-qa-efficiency:
+    requires_mc: yes
+    dependencies: [o2-analysis-event-selection, o2-analysis-trackselection_runX]
+
+  o2-analysis-qa-event-track:
+    requires_mc: yes
+    dependencies: [o2-analysis-event-selection, o2-analysis-trackselection_runX, o2-analysis-pid-tof-base]
+
+  # Helper tasks
+
+  o2-analysis-timestamp: {}
+
```

Comment:
Helper task block should go after the analysis block. Please check the ordering in codeHF.